library/std/src/pipe.rs

```diff
@@ -1,18 +1,26 @@
 //!  A cross-platform anonymous pipe.
 //!
-//! This module provides support for anonymous OS pipes, like [pipe] on Linux or [CreatePipe] on
-//! Windows.
+//! This module provides support for anonymous OS pipes, like [pipe] on UNIX/Linux or [CreatePipe]
```

Comment:
I think we can consider Linux to be a UNIX variant for all intents and purposes here
```suggestion
//! This module provides support for anonymous OS pipes, like [pipe] on UNIX or [CreatePipe]
```

Reply:
*shrug*, sure.